### PR TITLE
AFFINITY-69 UI: Add an About page that lists the version history changes

### PR DESCRIPTION
**Pull request by arthurwirsching**
We want to document the changes that have taken place in the application.
Here are a few examples
[http://www.csun.edu/faculty/about/version-history]
[http://www.csun.edu/faculty/beta/scholarship/about/version-history]

This task is done when
* We have a version history about page

### CODE REVIEW

File: resources/views/pages/about/version-history.blade.php [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+	<title>Affinity Web Service</title>
+	<meta name="description" content="A Web Service that delivers information on faculty engagement">
+	<link rel="icon" href="//www.csun.edu/sites/default/themes/csun/favicon.ico" type="image/x-icon">
+	<script src="//use.typekit.net/gfb2mjm.js"></script>
+	<script>try{Typekit.load();}catch(e){}</script>
+	<link rel="stylesheet" href="//fonts.googleapis.com/css?family=Open+Sans:300,300italic,400,400italic,600,600italic,700,700italic,800,800italic">
+	<link rel="stylesheet" href="{{ url('css/metaphor.css') }}">
+	<link rel="stylesheet" href="{{ url('css/tomorrow.css.min') }}">
+	<style>
```
Comment (luisjg) [inline resources/views/pages/about/version-history.blade.php]: We don't need lines 16 though 55

File: resources/views/pages/about/version-history.blade.php [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+				</ul>
+				<a class="header--sm" href="{{ url('/about/version-history') }}"><strong>Version History</strong></a>
+
+			</div>
+
+			<div class="col-md-9">
+				<div class="container">
+					<div class="row">
+						<div class="col-sm-12">
+							<h1 class="page-header">Version History</h1>
```
Comment (luisjg) [inline resources/views/pages/about/version-history.blade.php]: This class should be ```type--header type--thin```

File: resources/views/pages/about/version-history.blade.php [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+						</div>
+						<div class="col-sm-2">
+
```
Comment (luisjg) [inline resources/views/pages/about/version-history.blade.php]: Needs a bit of restructuring.

File: resources/views/pages/about/version-history.blade.php [hunk 4]
```diff
@@ -62,56 +23,63 @@
...
 				<a class="header--sm" href="{{ url('/') }}"><strong>DOCUMENTATION</strong></a>
 				<ul class="nav">
-					<li class="nav__item"><a class="nav__link" href="{{url('/#introduction')}}">Introduction</a></li>
-					<li class="nav__item"><a class="nav__link" href="{{url('/#getting-started')}}">Getting Started</a></li>
-					<li class="nav__item"><a class="nav__link" href="{{url('/#collections')}}">Collections</a></li>
-					<li class="nav__item"><a class="nav__link" href="{{url('/#subcollections')}}">Subcollections</a></li>
+					<li class="nav__item"><a class="nav__link" href="#introduction">Introduction</a></li>
+					<li class="nav__item"><a class="nav__link" href="#getting-started">Getting Started</a></li>
+					<li class="nav__item"><a class="nav__link" href="#collections">Collections</a></li>
+					<li class="nav__item"><a class="nav__link" href="#subcollections">Subcollections</a></li>
```
Comment (arthurwirsching) [inline resources/views/pages/about/version-history.blade.php]: This needs to go back to how it was previously.

**Comment by alejandrom2** [comment]
👍

**Pull request merged**